### DISCUSSION TITLE
ipv6 snap ubuntu install

### DISCUSSION THREAD

**Discussion by gnade** [sct/overseerr#3463] [Help]
Hey there, 

TLDR: New install of Overseer using snap on Ubuntu only seems to run using ipv6.

Trying to hit the website for a new Overseerr install on Ubuntu using snap but it looks like it has only activated ipv6. 
Used the simple command sudo snap install overseer to install it. 

It installed ok, no errors. However, I cannot hit the website on 5055. The log shows its running happily and waiting for config. 

Did a bit of investigating (not a linux guy really). As far as I can tell, the new installation of Overseerr is listening on 5055 using only ipv6.  As it's a virtual machine, I was able to replicate on a brand new ubuntu install, same thing. 

Listening Ports:  looks like "node" is overseerr, the 8181 there is tautulli which is working fine. 
sshd       724            root    3u  IPv4  20181      0t0  TCP *:22 (LISTEN)
sshd       724            root    4u  IPv6  20183      0t0  TCP *:22 (LISTEN)
node      1349            root   24u  IPv6  25635      0t0  TCP *:5055 (LISTEN)
python3   1710            root    7u  IPv4  25471      0t0  TCP *:8181 (LISTEN)

sudo snap connections overseerr
Interface     Plug                    Slot           Notes
home          overseerr:home          :home          -
network       overseerr:network       :network       -
network-bind  overseerr:network-bind  :network-bind  -

I did a bit of searching and cannot find this anywhere else, also could not find a way to configure the snap to only use ipv4. 

Distributor ID: Ubuntu
Description:    Ubuntu 22.04.2 LTS
Release:        22.04
Codename:       jammy

Has anyone experienced this before, or got any tips for resolving it?

Thanks
